src/componentLibrary/CoinIcon/icons/DotIcon.tsx

```diff
@@ -0,0 +1,16 @@
+const DotIcon = (): JSX.Element => (
+  <svg role='image' width='100%' viewBox='0 0 45 45' fill='none' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
One question, why not we are importing SVGs via SVGR? According to create-react-app official docs, it's a recommended way I'd say.

Reply:
@crypto-engineer won't need a separate ticket (it will be done when we replace the icons) but I'll create a placeholder for that now so that we don't lose track of this. Could you point me to an example of it in use in the dapp so that I can add that to the ticket as well?